server/workflow/workflow_server.go

```diff
@@ -210,13 +211,7 @@ func (s *workflowServer) WatchWorkflows(req *workflowpkg.WatchWorkflowsRequest,
 			return nil
 		case event, open := <-watch.ResultChan():
 			if !open {
-				log.Debug("Re-establishing workflow watch")
```

Comment:
as the client must have re-connection logic, we should not try and mask/mitigate the problem here